### DISCUSSION TITLE
راه اندازی ادبلاکر برای کاربران iOS بر روی Shadowrocket

### DISCUSSION THREAD

**Discussion by iam4lucard** [MasterKia/PersianBlocker#95] [Ideas]
سلام به همه
ممنون از زحمات همه دوستان.
من دیدم که نوشته بودین برای کاربران iOS راه حلی وجود نداره اما من خودم برای مدیریت ترافیک روی گوشی از اپلیکیشنی استفاده می‌کنم به اسم Shadowrocket که یه فایل کانفیگ داره که داخلش میشه هر نوع rule رو مشخص کرد و به صورت کامل گفت که چه چیزهایی لود بشه و چه چیزهایی لود نشه و از چه طریقی این اتفاق بیافته.
به نظرم میشه برای کاربران iOS یه فایل CONFIG درست کرد با توجه به اینکه تعداد کاربرانش هم هر روز داره افزایش پیدا می‌کنه.
من خودم به صورت دستی تعدادی از سرورها و دامین‌های تبلیغاتی رو داخلش اضافه کردم ولی نتونستم (چون فنی نیستم) لیست کامل شما رو اضافه بکنم.

**Comment by MasterKia** [comment]
درود بر شما.
بله نوشته بودم برای کاربران آیفون راهی وجود نداره.
اما برای کسانی که Shadowrocket دارن یک راهکار گذاشته بودم.

https://github.com/SamadiPour/iran-hosted-domains/blob/main/README.fa.md

https://github.com/SamadiPour/iran-hosted-domains/releases/latest

![Screenshot_20221205_125056](https://user-images.githubusercontent.com/17685483/205600532-0cf46a3f-cd04-4f5e-ae06-d95e47e16344.png)


لطفاً آزمایش کنید و اگر کارساز بود بگید تا به README بخش آیفون اضافه کنم که کاربران آیفون می‌تونن از نرم‌افزار Shadowrocket بهره ببرن.
